### PR TITLE
Add docs for `HTMLLinkElement.integrity` and `HTMLScriptElement.integrity`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Related issues and pull requests

part of the https://github.com/mdn/mdn/issues/520

### CODE REVIEW

File: files/en-us/web/api/htmllinkelement/index.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ _Inherits properties from its parent, {{domxref("HTMLElement")}}._
   - : A string representing the URI for the target resource.
 - {{domxref("HTMLLinkElement.hreflang")}}
   - : A string representing the language code for the linked resource.
+- {{domxref("HTMLLinkElement.integrity")}}
+  - : A string that reflects the inline metadata that user agent can use to verify that a fetched resource has been delivered free of unexpected manipulation. It reflects the `integrity` attribute of the {{HTMLElement("link")}} element.
```
Comment (dawei-wang) [inline files/en-us/web/api/htmllinkelement/index.md:31]:
```suggestion
  - : A string that reflects the inline metadata that a user agent can use to verify that a fetched resource has been delivered free of unexpected manipulation. It reflects the `integrity` attribute of the {{HTMLElement("link")}} element.
```

File: files/en-us/web/api/htmllinkelement/integrity/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+title: "HTMLLinkElement: integrity property"
+short-title: integrity
+slug: Web/API/HTMLLinkElement/integrity
+page-type: web-api-instance-property
+browser-compat: api.HTMLLinkElement.integrity
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`integrity`** property of the {{domxref("HTMLLinkElement")}} interface is a string that reflects the inline metadata that user agent can use to verify that a fetched resource has been delivered free of unexpected manipulation.
```
Comment (dawei-wang) [inline files/en-us/web/api/htmllinkelement/integrity/index.md:11]:
```suggestion
The **`integrity`** property of the {{domxref("HTMLLinkElement")}} interface is a string that reflects the inline metadata that a user agent can use to verify that a fetched resource has been delivered free of unexpected manipulation.
```

File: files/en-us/web/api/htmlscriptelement/index.md [hunk 3]
```diff
@@ -39,6 +39,8 @@ _Inherits properties from its parent, {{domxref("HTMLElement")}}._
   - : A string; an obsolete way of registering event handlers on elements in an HTML document.
 - {{domxref("HTMLScriptElement.fetchPriority")}}
   - : An optional string representing a hint given to the browser on how it should prioritize fetching of an external script relative to other external scripts. If this value is provided, it must be one of the possible permitted values: `high` to fetch at a high priority, `low` to fetch at a low priority, or `auto` to indicate no preference (which is the default). It reflects the `fetchpriority` attribute of the {{HTMLElement("script")}} element.
+- {{domxref("HTMLScriptElement.integrity")}}
+  - : A string that reflects the inline metadata that user agent can use to verify that a fetched resource has been delivered free of unexpected manipulation. It reflects the `integrity` attribute of the {{HTMLElement("script")}} element.
```
Comment (dawei-wang) [inline files/en-us/web/api/htmlscriptelement/index.md:43]:
```suggestion
  - : A string that reflects the inline metadata that a user agent can use to verify that a fetched resource has been delivered free of unexpected manipulation. It reflects the `integrity` attribute of the {{HTMLElement("script")}} element.
```

File: files/en-us/web/api/htmlscriptelement/integrity/index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+title: "HTMLScriptElement: integrity property"
+short-title: integrity
+slug: Web/API/HTMLScriptElement/integrity
+page-type: web-api-instance-property
+browser-compat: api.HTMLScriptElement.integrity
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`integrity`** property of the {{domxref("HTMLScriptElement")}} interface is a string that reflects the inline metadata that user agent can use to verify that a fetched resource has been delivered free of unexpected manipulation.
```
Comment (dawei-wang) [inline files/en-us/web/api/htmlscriptelement/integrity/index.md:11]:
```suggestion
The **`integrity`** property of the {{domxref("HTMLScriptElement")}} interface is a string that reflects the inline metadata that a user agent can use to verify that a fetched resource has been delivered free of unexpected manipulation.
```

**Review by dawei-wang — COMMENTED** [review]
add missing word

File: files/en-us/web/api/htmllinkelement/index.md [hunk 5]
```diff
@@ -27,6 +27,8 @@ _Inherits properties from its parent, {{domxref("HTMLElement")}}._
   - : A string representing the URI for the target resource.
 - {{domxref("HTMLLinkElement.hreflang")}}
   - : A string representing the language code for the linked resource.
+- {{domxref("HTMLLinkElement.integrity")}}
+  - : A string that reflects the inline metadata that a user agent can use to verify that a fetched resource has been delivered free of unexpected manipulation. It reflects the `integrity` attribute of the {{HTMLElement("link")}} element.
```
Comment (teoli2003) [inline files/en-us/web/api/htmllinkelement/index.md:31]: We prefer _browser_.
```suggestion
  - : A string that contains inline metadata that a browser can use to verify that a fetched resource has been delivered without unexpected manipulation. It reflects the `integrity` attribute of the {{HTMLElement("link")}} element.
```

File: files/en-us/web/api/htmllinkelement/integrity/index.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+title: "HTMLLinkElement: integrity property"
+short-title: integrity
+slug: Web/API/HTMLLinkElement/integrity
+page-type: web-api-instance-property
+browser-compat: api.HTMLLinkElement.integrity
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`integrity`** property of the {{domxref("HTMLLinkElement")}} interface is a string that reflects the inline metadata that a user agent can use to verify that a fetched resource has been delivered free of unexpected manipulation.
```
Comment (teoli2003) [inline files/en-us/web/api/htmllinkelement/integrity/index.md:11]:
```suggestion
The **`integrity`** property of the {{domxref("HTMLLinkElement")}} interface is a string containing inline metadata that a browser can use to verify that a fetched resource has been delivered without unexpected manipulation.
```

File: files/en-us/web/api/htmlscriptelement/index.md [hunk 7]
```diff
@@ -39,6 +39,8 @@ _Inherits properties from its parent, {{domxref("HTMLElement")}}._
   - : A string; an obsolete way of registering event handlers on elements in an HTML document.
 - {{domxref("HTMLScriptElement.fetchPriority")}}
   - : An optional string representing a hint given to the browser on how it should prioritize fetching of an external script relative to other external scripts. If this value is provided, it must be one of the possible permitted values: `high` to fetch at a high priority, `low` to fetch at a low priority, or `auto` to indicate no preference (which is the default). It reflects the `fetchpriority` attribute of the {{HTMLElement("script")}} element.
+- {{domxref("HTMLScriptElement.integrity")}}
+  - : A string that reflects the inline metadata that a user agent can use to verify that a fetched resource has been delivered free of unexpected manipulation. It reflects the `integrity` attribute of the {{HTMLElement("script")}} element.
```
Comment (teoli2003) [inline files/en-us/web/api/htmlscriptelement/index.md:43]:
```suggestion
  - : A string that contains inline metadata that a browser can use to verify that a fetched resource has been delivered without unexpected manipulation. It reflects the `integrity` attribute of the {{HTMLElement("script")}} element.
```

File: files/en-us/web/api/htmlscriptelement/integrity/index.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+title: "HTMLScriptElement: integrity property"
+short-title: integrity
+slug: Web/API/HTMLScriptElement/integrity
+page-type: web-api-instance-property
+browser-compat: api.HTMLScriptElement.integrity
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`integrity`** property of the {{domxref("HTMLScriptElement")}} interface is a string that reflects the inline metadata that a user agent can use to verify that a fetched resource has been delivered free of unexpected manipulation.
```
Comment (teoli2003) [inline files/en-us/web/api/htmlscriptelement/integrity/index.md:11]:
```suggestion
The **`integrity`** property of the {{domxref("HTMLScriptElement")}} interface is a string that contains inline metadata that a browser can use to verify that a fetched resource has been delivered without unexpected manipulation.
```

File: files/en-us/web/html/element/script/index.md [hunk 9]
```diff
@@ -58,7 +58,7 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
       - : Default: Signals automatic determination of fetch priority relative to other external scripts.
 
 - `integrity`
-  - : This attribute contains inline metadata that a user agent can use to verify that a fetched resource has been delivered free of unexpected manipulation. See [Subresource Integrity](/en-US/docs/Web/Security/Subresource_Integrity).
+  - : This attribute contains inline metadata that a user agent can use to verify that a fetched resource has been delivered free of unexpected manipulation. The attribute must not specified when the `src` attribute is not specified. See [Subresource Integrity](/en-US/docs/Web/Security/Subresource_Integrity).
```
Comment (teoli2003) [inline files/en-us/web/html/element/script/index.md:61]:
```suggestion
  - : This attribute contains inline metadata that a user agent can use to verify that a fetched resource has been delivered without unexpected manipulation. The attribute must not specified when the `src` attribute is not specified. See [Subresource Integrity](/en-US/docs/Web/Security/Subresource_Integrity).
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A few minor change and this can be merged! Thank you!

File: files/en-us/web/html/element/link/index.md [hunk 10]
```diff
@@ -224,6 +224,7 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
 - `integrity`
   - : Contains inline metadata — a base64-encoded cryptographic hash of the resource (file) you're telling the browser to fetch.
     The browser can use this to verify that the fetched resource has been delivered free of unexpected manipulation.
```
Comment (skyclouds2001) [inline files/en-us/web/html/element/link/index.md:226]:
```suggestion
    The browser can use this to verify that the fetched resource has been delivered without unexpected manipulation.
```

**Review by teoli2003 — APPROVED** [review]
👍 Thank you!

**Pull request merged**